test/chart/BarChart.spec.tsx

```diff
@@ -599,27 +599,15 @@ describe('<BarChart />', () => {
     });
 
     test('Stacked bars are actually stacked', () => {
-      let seriesOneBarOneValue, seriesTwoBarOneValue;
-      const Spy = (props: { formattedGraphicalItems?: any }) => {
```

Comment:
Can we replace this with selectors?

Reply:
Maybe but I've been having trouble with the selectors in tests

Reply:
done